content/how-to-todo-in-ember-template-lint.md

```diff
@@ -0,0 +1,121 @@
+---
+title: How To Todo In Ember Template Lint
+authors:
+  - melanie-sumner
+date: 2021-02-02T00:00:00.000Z
+tags:
+  - addons
+  - '2021'
+---
+
+## Introduction
+
+**At the time of this writing, this feature is available in `ember-template-lint@3.0.0-beta.3`.**
+
+The new `todo` feature in the `ember-template-lint` addon provides a new option to the existing states for linting rules.
+
+With this new functionality, linting issues that are found can be converted into a `todo`, giving developers the ability to strategically plan the work to resolve issues in existing code. A `todo`, can be set to automatically become a `warning` and then an `error` with the use of a supporting feature called decay days.
+
+<img src="/images/blog/how-to-todo-in-ember-template-lint/todo-warning-error-graphic.png" alt="todo, warning, error">
+
+Previously, running the  `lint:hbs` script (default in an Ember app) would cause `ember-template-lint` to run on all of the `.hbs` files, and return a result that included number of `warnings` and number of `errors` found.
+<img src="/images/blog/how-to-todo-in-ember-template-lint/terminal-without-todo.png" alt="a terminal showing results that include 2 errors and 0 warnings" />
+
+Now, however, we have a new option- we can also include number of `todos`.  After updating the  `lint:hbs` script in the `package.json` file to use the `--include-todo` flag (we want to be reminded that we have todos!), like this:
+
+```json
+"lint:hbs": "ember-template-lint . --include-todo",
+```
+
+Run the command (`yarn lint:hbs`) again:
+<img src="/images/blog/how-to-todo-in-ember-template-lint/terminal-with-todo.png" alt="a terminal showing 2 errors, 0 warnings, and 0 todos" />
+
+Note: `ember-template-lint` also takes a `--quiet` flag, which won't print any warnings or todos, only errors.
+
+Along with the ability to create todos out of currently failing rules, some options can be set to decide the number of days before the `todo` turns into a `warning` and then into an `error`.
+
+### Why, though?
+
+This new feature is really intended to help apps at scale. It can be tricky to update a dependency like `ember-template-lint`, especially if it's a version that includes a new rule, due to the number of factors involved. Taking in a new version of the linter usually means a lot of communication and coordination about the new rule(s) and the larger your app gets, and the more developers you have, the more work this was. For global companies that use Ember, there might even be a committee whose sole focus is to make decisions about linting rules. That's a lot of red tape!
```

Comment:
I like to avoid adverbs that are hard to quantify and may sound sensational. How does the following change sound to you?

```suggestion
This new feature is intended to help apps at a large scale. It can be tricky to update a dependency like `ember-template-lint` when it includes a new rule, due to a number of factors involved. Taking in a new version of the linter can mean communication and coordination about the new rule(s). The larger your app is and the more developers you have, the more work this can be. For global companies that use Ember, there might even be a committee whose sole focus is to make decisions about linting rules!
```